dbschema/location.esdl

```diff
@@ -0,0 +1,28 @@
+module default {
+  abstract type Location extending Resource {
+    required name: str {
```

Comment:
It's grab this from Named from the PR I just put up.
https://github.com/SeedCompany/cord-api-v3/pull/2933/files#diff-038ac5fc7d17f701e97927ce062fb044c9dd954bdbbf3566134674f04bb87194L2-R3